projects/github.com/helmfile/vals/package.yml

```diff
@@ -0,0 +1,31 @@
+distributable:
+  url: https://github.com/helmfile/vals/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+display-name: pinniped
```

Comment:
this is from the prior PR, right? ;)

Reply:
Hehe nice catch thanks!

Reply:
I removed it, I guess by default the last component of the package path is used, right ?

Reply:
For the pinniped PKG, I mixed the sample https://github.com/pkgxdev/brewkit/blob/main/share/TEMPLATE.pkg.yml and some other packages manifests. However, this field is not present in the sample. It is used somewhere ?

Reply:
> I removed it, I guess by default the last component of the package path is used, right ?

yup.

> However, this field is not present in the sample. It is used somewhere ?

initially it was for the ossapp gui, but it is also used in short-form commands. we should add it to the template.

Reply:
https://github.com/pkgxdev/brewkit/issues/183